<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="test_button">
      <property name="text">
       <string>button</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="test_checkbox">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="test_combobox"/>
    </item>
    <item>
     <widget class="QLabel" name="test_label">
      <property name="text">
       <string>label</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="test_lineedit"/>
    </item>
    <item>
     <widget class="QSlider" name="test_slider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="test_spinbox"/>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="test_textarea"/>
    </item>
    <item>
     <widget class="QTreeWidget" name="test_treeview">
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string>asdf</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>asdf2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>asdf3</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
